dts/riscv/espressif/esp32c6/esp32c6_lpcore.dtsi

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2024 Espressif Systems (Shanghai) Co., Ltd.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <mem.h>
+#include <freq.h>
+#include <zephyr/dt-bindings/gpio/gpio.h>
+#include <zephyr/dt-bindings/interrupt-controller/esp-esp32c6-intmux.h>
+#include <zephyr/dt-bindings/clock/esp32c6_clock.h>
+#include <dt-bindings/pinctrl/esp32c6-pinctrl.h>
+
+/ {
+	#address-cells = <1>;
+	#size-cells = <1>;
+
+	cpus {
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		cpu0: cpu@0 {
+			device_type = "cpu";
+			compatible = "espressif,riscv";
+			riscv,isa = "rv32imac_zicsr_zifencei";
+			reg = <0>;
+			clock-source = <ESP32_RTC_FAST_CLK_SRC_XTAL_D2>;
+			clock-frequency = <DT_FREQ_M(20)>;
+			xtal-freq = <DT_FREQ_M(40)>;
+		};
```

Comment:
Why not adding lpcore cpu under the `cpus` node?

Reply:
What do you mean? `cpu0` is lpcore in this context

Reply:
you're right. I was tricked by `cpu0:spu@0`, shouldn't be this numbered otherwise?